cmd/virt-handler/Dockerfile

```diff
@@ -23,9 +23,20 @@ FROM centos:7
 # no-lower than what was used when the binary was built.
 RUN yum -y install wget && \
     wget -P /etc/yum.repos.d https://copr.fedorainfracloud.org/coprs/jmliger/virt7-upstream/repo/epel-7/jmliger-virt7-upstream-epel-7.repo  \
-    && yum -y install libvirt-client \
+    && yum -y install libvirt-client cyrus-sasl-md5 \
     && yum -y clean all
 
+COPY auth.conf /etc/libvirt/auth.conf
 COPY virt-handler /virt-handler
+COPY virt-handler.sh /virt-handler.sh
+RUN chmod a+x /virt-handler.sh
```

Comment:
Every one of these RUN commands creates a separate layer. By combining them into a single RUN execution, we can collapse all of these layers into one.  Reducing layers helps with performance.

```
RUN cmd1 && \
   cmd2 && 
   cmd3 && \
   ect...
```